MultisigIntegrationTests/CoreData/NetworkTests.swift

```diff
@@ -230,19 +229,19 @@ class NetworkTests: CoreDataTestCase {
         XCTAssertEqual(networkSafes[0].safes[1].name, "22")
         XCTAssertEqual(networkSafes[0].safes[2].name, "20")
 
-        XCTAssertEqual(networkSafes[1].network, network1)
-        XCTAssertEqual(networkSafes[1].safes.count, 2)
-        XCTAssertEqual(networkSafes[1].safes[0].name, "01")
-        XCTAssertEqual(networkSafes[1].safes[1].name, "00")
+        XCTAssertEqual(networkSafes[2].network, network1)
```

Comment:
Why the logic was changed here?

Reply:
The unit testing was failing with out of range exception